packages/account/src/Components/Routes/helpers.ts

```diff
@@ -10,15 +10,15 @@ export const findRouteByPath = (path: string, routes_config?: TRouteConfig[]): R
     routes_config?.some(route_info => {
         let match_path: match | null = null;
         try {
-            match_path = matchPath(path, route_info);
+            match_path = matchPath(path, route_info as RouteProps);
```

Comment:
Why we need this type assertion?

Reply:
type of route_info TRouteConfig is not compatible with the expected type RouteProps.
since TRouteConfig is used in multiple places, maybe for now, we can do this..